# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/apz/src/GestureEventListener.h
# Commit: 1d9384906287
# Full Hash: 1d93849062870a8d492eeef2150ecea174cb6223
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-05-26 21:12:53
# Regressor Bug: 1550422
# File Overlap Count: 1
# Description:
#   Bug 1550422 - P15. Move Skip and Once gfxPrefs to StaticPrefs. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31259
# ==============================================================================

diff -r f5ebdcca640d -r 1d9384906287 gfx/layers/apz/src/GestureEventListener.h
--- a/gfx/layers/apz/src/GestureEventListener.h	Sun May 26 14:29:56 2019 +0000
+++ b/gfx/layers/apz/src/GestureEventListener.h	Sun May 26 14:30:14 2019 +0000
@@ -84,7 +84,7 @@
 
     // A touch start with a single touch point has just happened.
     // After having gotten into this state we start timers for MAX_TAP_TIME and
-    // gfxPrefs::UiClickHoldContextMenusDelay().
+    // StaticPrefs::UiClickHoldContextMenusDelay().
     // Allowed next states: GESTURE_MULTI_TOUCH_DOWN, GESTURE_NONE,
     //                      GESTURE_FIRST_SINGLE_TOUCH_UP,
     //                      GESTURE_LONG_TOUCH_DOWN,
@@ -93,8 +93,8 @@
 
     // While in GESTURE_FIRST_SINGLE_TOUCH_DOWN state a MAX_TAP_TIME timer got
     // triggered. Now we'll trigger either a single tap if a user lifts her
-    // finger or a long tap if gfxPrefs::UiClickHoldContextMenusDelay() happens
-    // first.
+    // finger or a long tap if StaticPrefs::UiClickHoldContextMenusDelay()
+    // happens first.
     // Allowed next states: GESTURE_MULTI_TOUCH_DOWN, GESTURE_NONE,
     //                      GESTURE_LONG_TOUCH_DOWN.
     GESTURE_FIRST_SINGLE_TOUCH_MAX_TAP_DOWN,